hack/lib/install.sh

```diff
@@ -98,11 +98,55 @@ verify_docker_installed(){
 function install_cni_plugins() {
   # install CNI plugins if not exist
   if [ ! -f "/opt/cni/bin/loopback" ]; then
-    echo -e "install CNI plugins..."
+    echo -e "installing CNI plugins..."
     mkdir -p /opt/cni/bin
-    wget https://github.com/containernetworking/plugins/releases/download/v1.1.1/cni-plugins-linux-amd64-v1.1.1.tgz
-    tar Cxzvf /opt/cni/bin cni-plugins-linux-amd64-v1.1.1.tgz
+    wget $CNI_DOWNLOAD_ADDR
+    tar Cxzvf /opt/cni/bin ${CNI_DOWNLOAD_ADDR##*/}
+    # remove installation package after the installation is complete
+    rm -rf ${KUBEEDGE_ROOT}/${CNI_DOWNLOAD_ADDR##*/}
   else
     echo "CNI plugins already installed and no need to install"
   fi
+
+  # create CNI netconf file if not exist
+  CNI_DIR="/etc/cni/net.d"
+  if [ "`ls -A $CNI_DIR`" = "" ]; then
```

Comment:
if this directory not exist, it will also NOT create netconf file, so better add a `mkdir -p /etc/cni/net.d`

Reply:
maybe just check whether file `/etc/cni/net.d/10-containerd-net.conflist` exist or not

Reply:
Done, when CNI_DIR not exist,  it will also print `ls: cannot access '/etc/cni/net.d': No such file or directory`

Reply:
Seems this file exists in node of github action, the local-up can run in CI :)

Reply:
on my local machine, this file not exists, but it seems run well 😂

Reply:
add sudo?